astropy/coordinates/builtin_frames/galactocentric.py

```diff
@@ -41,13 +43,21 @@ class galactocentric_frame_defaults(ScienceState):
     ``Galactocentric`` or ``Galactocentric()`` with no explicit arguments.
 
     This class controls the parameter settings by specifying a string name,
-    which can be one of:
+    which has the pre-specified options:
```

Comment:
maybe "the following pre-specified options"

Reply:
- [ ] implemented